src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -300,6 +338,21 @@ public void SetCheckpointDir(string directory)
             _jvmObject.Invoke("setCheckpointDir", directory);
         }
 
+        /// <summary>
+        /// Return the directory where RDDs are checkpointed.
+        /// </summary>
+        /// <returns>
+        /// The directory where RDDs are checkpointed. Returns `null` if no checkpoint
+        /// directory has been set.
+        /// </returns>
+        public string GetCheckpointDir()
+        {
+            var optionalCheckPointDir =
+                new Option((JvmObjectReference)_jvmObject.Invoke("getCheckpointDir"));
+
+            return optionalCheckPointDir.IsDefined() ? (string)optionalCheckPointDir.Get() : null;
```

Comment:
`OrNull()`?

Btw, is this how we have been handling if the API returns `Option`?

Reply:
yup, we've been using the above pattern in other places.